test/lib/event_daemon/monitor_test.rb

```diff
@@ -39,7 +39,33 @@ def test_monitor_not_running
 
         MockService.expects(:run)
 
-        monitor.check_services(nil, nil)
+        monitor.check_services
+      end
+
+      def test_monitor_starting
+        Rails.cache.expects(:write).with(
```

Comment:
I thought making a test double of `write` would be more correct than reading the "real" Rails cache, since this test isn't trying to test `Rails.cache`.

Reply:
For some reason I have a rule in my head "mock whatever you're not testing" so that's how I was thinking.

Reply:
Works for me